src/config/default-config.js

```diff
@@ -19,20 +19,30 @@ const GAMES = require('../constants/games');
 const Disk = require('../utils/disk');
 
 export default class DefaultConfig {
-  constructor() {
+  constructor({
+      username="",
+      user0="sculpture0",
+      user1="sculpture1",
+      user2="sculpture2"
+  } = {}) {
```

Comment:
Note the `= {}` this technique is a weird quirk that is absolutely necessary if you want to be able to pass no object into this constructor. It doesn't matter if every argument has a default value, you still need `= {}` to make the parameter optional. Causes some really weird errors if you leave that out.